src/pages/home/home.ts

```diff
@@ -26,23 +26,32 @@ export class HomePage {
       public walletSrv: WalletsService,
       public currencySrv: CurrencyService,
       public http: Http,
-  ) {}
+      public loadingCtrl: LoadingController,
+  ) {
+    this.loader = this.loadingCtrl.create({
+      content: "Loggin in.",
```

Comment:
Change this to 'Loading' or something similar instead of 'Loggin in'. 
1) Loggin in is not very professional because of the typo
2) Homepage is also reloaded when withdrawing, deposit, etc in addition to logging in